src/main/java/com/dev/cinema/model/dto/order/OrderResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.dev.cinema.model.dto.order;
+
+import com.dev.cinema.model.dto.ticket.TicketResponseDto;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class OrderResponseDto {
+    private Long id;
+    private LocalDateTime orderDate;
+    private List<TicketResponseDto> tickets;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
this may make the life easier
what do you think?

Reply:
Я представила как должен выглядеть список всех заказов на фронте, и мне показалось, что выводить инфу только про номер билета и дату заказа не информативно, но можно и так